### ISSUE TITLE
Why my loss doesn't go down past 0.69

### ISSUE THREAD

**Issue by LetuGitHub** [Qucy/Siamese-Network#1]
Hello @Qucy:
I'm trying to implement your project using PyTorch. But when I started to train my own datasets, the loss doesn't go down past 0.69.
One thing worth noting is that I can achieve normal loss reduction when training this dataset using VGG16 + Siamese. Do I need to incorporate the pre-trained ResNet model?
